ifrs17/Utils/EqualityComparers.ipynb

```diff
@@ -116,7 +116,8 @@
                 "\n    private YieldCurveComparer(){}",
                 "\n",
                 "\n\tpublic bool Equals(YieldCurve x, YieldCurve y) => ",
-                "\n        x.Year == y.Year && x.Month == y.Month && x.Scenario == y.Scenario && x.Currency == y.Currency && x.Id == y.Id && x.Values.SequenceEqual(y.Values, Precision);",
+                "\n        x.Year == y.Year && x.Month == y.Month && x.Scenario == y.Scenario && x.Currency == y.Currency ",
+                "\n        && x.Id == y.Id && x.Name == y.Name && x.Values.SequenceEqual(y.Values, Precision);",
```

Comment:
I don't understand why you have the Values here, as this is not part of the identity

Reply:
Comparing the values are used to check if the current import contains information already present in the DB. 
This is needed when Yield Curve importer computes IfrsVariable, which will be implemented in the next PR.